packages/settings-library/src/settings_library/docker_registry.py

```diff
@@ -24,7 +25,7 @@ class RegistrySettings(BaseCustomSettings):
 
     @validator("REGISTRY_PATH", pre=True)
     @classmethod
-    def escape_none_string(cls, v):
+    def escape_none_string(cls, v) -> Any | None:
         return None if v == "None" else v
```

Comment:
MINOR: I think the string `"None"` or `"null"` are parsed as `None`. Did you try reverting the order  `None | str`? 
In any case, it is subtle so I would add a test to ensure nobody messes it up later :-)